object-mapper.rst

```diff
@@ -0,0 +1,259 @@
+Object Mapping
+==============
+
+Symfony provides a mapper to transform a given object to another one.
+
+.. _activating_the_serializer:
+
+Installation
+------------
+
+Run this command to install the ``object-mapper`` before using it:
+
+.. code-block:: terminal
+
+    $ composer require symfony/object-mapper
+
+Using the ObjectMapper Service
+------------------------------
+
+Once installed, the object mapper service can be injected in any service where
+you need it or it can be used in a controller::
+
+    // src/Controller/DefaultController.php
+    namespace App\Controller;
+
+    use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+    use Symfony\Component\HttpFoundation\Response;
+    use Symfony\Component\ObjectMapper\ObjectMapperInterface;
+
+    class DefaultController extends AbstractController
+    {
+        public function index(ObjectMapperInterface $objectMapper): Response
+        {
+            // keep reading for usage examples
+        }
+    }
+
+
+Map an object to another one
+----------------------------
+
+To map an object to another one use ``map``::
+
+    use App\Dto\Source;
+    use App\Dto\Target;
+
+    $mapper = new ObjectMapper();
+    $mapper->map(source: new Source(), target: Target::class);
+
+
+If you alread have a target object, you can use its instance directly::
```

Comment:
```suggestion
If you already have a target object, you can use its instance directly::
```

Small typo. Not related question, can we expect APP with ObjectMapper DTO support or it's just works now?

Also, this probably needs to target 7.3.

Thanks!